api/src/utils/hooks.ts

```diff
@@ -52,33 +48,19 @@ export async function verifyJWT(context: HookContext) {
     return Promise.reject(new Forbidden("The authorization header is missing"));
   }
   const token = authHeader.split(" ")[1];
```

Comment:
Should the extract function be used here too?

Reply:
Good catch, I missed that one.